examples/events/snowplow-debugger/search-result-product.json

```diff
@@ -2,19 +2,23 @@
     "description": "Schema for Product Search Result Entity",
     "properties": {
       "name": {
-        "type": "string"
+        "type": "string",
+        "minLength": 1
```

Comment:
What happens here if the search returns no results? We might still need the events but with empty fields?